### ISSUE TITLE
Markdown cells with verbatim blocks get horizontal scroll bars.

### ISSUE THREAD

**Issue by IanSudbery** [damianavila/RISE#430]
I am loving using RISE in my undergrad python classes, but I have a problem:

Any markdown cell that has a verbatim block gets horizontal scroll bars in the slideshow. 

For example a cell:

```
This is a test

This is not a block
```

renders fine, but

```
This is a test:

    This is a block
```

Does get a horizontal scroll bar. The funny thing is, it doesn't matter how wide the broswer window is, the bar is always there. I can make the window half my screen, all my screen or zoom right out, but the bar is still there. 

I've tried both indented blocks and github style tripple backtick blocks, with the same result each time.

**Comment by CaptainProton42** [comment]
This seems to be caused by the \<pre> tag. Adding padding to every ``text_cell_render`` div via css
```
.reveal .text_cell_render {
	padding-right: 50px;
}
```
fixed the problem for me and didn't seem to mess up anything else.

**Comment by damianavila** [comment]
@CaptainProton42 PR incoming :wink:?

**Comment by damianavila** [comment]
Fixed by #459. Closing here.
